### ISSUE TITLE
kableextra fixed_thead

### ISSUE THREAD

**Issue by ghowoo** [juba/rmdformats#105]
I like rmdformats and use the readthedown theme a lot. With kableextra, I deliver tables in HTML format very well. There are two issues that bother me and if you give some tips to solve the problem.

One is the fixed_thead function from kableextra which holds the top row (row names) of the table on the screen when scrolling down the long table. The HTML file exported through rmdformats doesn't have this feature.

The other one is also related to the kableextra, the HTML table in the fmdformats exported file shows some vertical lines. But there should not be there if it is a HTML table just exported from kable/kableextra.

df <- data.frame(var1=1:24, var2=LETTERS[1:24])
df %>% kable() %>% kable_styling(fixed_thead=TRUE)

You can see the difference between the individual HTML table and the html table from the export through rmdformats.

Thanks for your help.

**Comment by juba** [comment]
Hi,

For the `fix_thead` function to work, you must fix the table wrapper maximum height. This can be done by adding some CSS code to your document : 

```html
<style>
#main .table-wrapper {
  max-height: 200px;
}
</style>
```

In the same way, if you want to remove vertical lines in tables you can do it with : 

```html
<style>
#main .table-wrapper table th,
#main .table-wrapper table td {
  border-left: none;
  border-right: none;
}
</style>
```

**Comment by ghowoo** [comment]
Thanks juba for the prompt response. The CSS code removes vertical lines, but the solution for fix_thead is not decent.

By fixed maximum height, the table has a fixed height with a scrollbar on the right side. It is not the desired effect of fixed_thead. By fixed_thead, no part of the table will be hidden except out of the window, and the head only hovers on top of the screen when the table has been rolled down to the end of a very long table that otherwise the head would not be seen on the top of the screen.

**Comment by juba** [comment]
Ah, my bad, thanks for the explanation, indeed I didn't understand the issue correctly.

I think this should now be fixed in the development version, but you'll still have to add a bit of custom CSS : 

```html
<style>
#main .table-wrapper {
  overflow-x: visible;
}
</style>
```

Sorry not to provide a more straightforward solution but I don't see for the moment how to have both sticky headers and horizontally scrollable tables.

**Comment by ghowoo** [comment]
Thanks. It works great with the sticky header. However, you somewhat just disabled the "hover" effect of kableextra. The background colors of table rows are fixed by your updated package. There should be a hover effect that when the mouse hovers on the row, the background color of the row would change a little bit. Your previous version of the package had the effect. This is one of the advantages to present table in HTML format rather than in Word or Latex format. Anyway, thank you very much for addressing the issue.

**Comment by ghowoo** [comment]
Here is the r code:
kable_styling(fixed_thead=TRUE, bootstrap_options = c("striped", "hover", "condensed", "responsive"))

**Comment by juba** [comment]
Yes, unfortunately it won't work because `kableExtra` relies on the bootstrap CSS for most of its functionalities, and this CSS is not included in `readthedown` as the template is not based on this framework. So for this you'll have to add your custom CSS to enable it.

```html
<style>
.table tbody tr:hover {
  background-color: #EEE;
}
</style>
```

**Comment by ghowoo** [comment]
Thanks. The CSS style works.
